drivers/sx127x/sx127x.c

```diff
@@ -95,17 +101,17 @@ int sx127x_reset(const sx127x_t *dev)
     gpio_set(dev->params.rx_switch_pin);
 #endif
 
-    /* Set reset pin to 0 */
-    gpio_clear(dev->params.reset_pin);
+    /* set reset pin to the state that triggers manual reset */
+    gpio_write(dev->params.reset_pin, SX127X_POR_ACTIVE_LOGIC_LEVEL);
 
-    /* Wait 1 ms */
-    xtimer_usleep(1000);
+    /* Wait for at least 100 µs */
+    xtimer_usleep(110);
 
     /* Put reset pin in High-Z */
     gpio_init(dev->params.reset_pin, GPIO_IN);
 
-    /* Wait 10 ms */
-    xtimer_usleep(1000 * 10);
+    /* Wait for at least 5 ms */
+    xtimer_usleep(1100 * 5);
```

Comment:
avoid magic numbers, and use conversion macros (i.e. `US_PER_MS`) here. To make this work simply wait 6ms (= `6 * US_PER_MS`)